packages/next-bundle-analyzer/index.js

```diff
@@ -1,17 +1,26 @@
 module.exports =
-  ({ enabled = true, openAnalyzer = true } = {}) =>
+  ({
+    enabled = true,
+    openAnalyzer = true,
+    analyzerMode = 'static',
+    serverReportFilename,
+    clientReportFilename,
+  } = {}) =>
```

Comment:
Is it possible to support all `BundleAnalyzerPlugin` options here (instead of a select few)?